tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -229,7 +229,8 @@ TEST_P(RGBToYUVTest, ConvertWholeRange) {
       static_cast<double>(diff_sum) / static_cast<double>(num_diffs);
   const double average_abs_diff =
       static_cast<double>(abs_diff_sum) / static_cast<double>(num_diffs);
-  const double psnr = GetPsnr(sq_diff_sum, num_diffs, rgb_max);
+  const double psnr = GetPsnr(static_cast<double>(sq_diff_sum),
+                              static_cast<double>(num_diffs), rgb_max);
```

Comment:
libavif\tests\gtest\avifrgbtoyuvtest.cc(232): warning C4244: 'argument': conversion from 'int64_t' to 'double', possible loss of data
libavif\tests\gtest\avifrgbtoyuvtest.cc(232): warning C4244: 'argument': conversion from 'int64_t' to 'double', possible loss of data

Reply:
Alternatively we can do the cast inside GetPsnr() instead at the call site, but this is good as is.